### ISSUE TITLE
a little confused about one line code in load_chunk.

### ISSUE THREAD

**Issue by Joefi** [mravanelli/pytorch-kaldi#161]
Hi, I'm a little confused about one line code in load_chunk.
`data_lab=data_lab-data_lab.min()`
Whatever the value of data_lab.min() is, the minimum value of elements in  data_lab will be zero after executing this line of code. if the value of data_lab.min() is not equal to zero, I think this line of code will lead to some problems.
For example, if data_lab.min() is equal to 5, it means the real label of the frame is 5, but the frame label will become 0 after executing this line of code.
look forward to your reply.

**Comment by mravanelli** [comment]
Hi,
that line ensures that all the labels are within 0 and N-1, as required by
the neural network. Note that the minimum label is computed over a very
large chunk (chunks should be composed of about 1 hour of speech) and in
such a big chunk you have for sure the actual minimum label. Moreover, in
the kaldi alignments, the minimum label is always "1" and that label
corresponds to silence that occurs very very frequently.

Best,

Mirco

**Comment by Joefi** [comment]
Thank you
